### PR TITLE
pre-alpha: SDK build and readme

### PR DESCRIPTION
**Pull request by Bars92**
## Description

- fix node sdk build to 2017, avoid unused dependencies such as `tslib`
- split the readme to docs
- updated core-js to minor
- export cookie names

### CODE REVIEW

**Comment by Bars92** [comment]
@jeff-descope please take a look at the language additions around oauth if you can

**Comment by meirwah** [comment]
@nirgur Can you review ? we need to deliver this tomorrow to a customer

**Review by nirgur — APPROVED** [review]
Looks good
I didn't review the docs

File: tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"rootDir": ".",
-		"target": "es6",
+		"target": "es2017",
```
Comment (dorsha) [inline tsconfig.json:4]: Make sure it works with node 12

**Pull request merged**